src/gc/gc.cpp

```diff
@@ -899,39 +893,69 @@ class t_join
         }
     }
 
+#ifdef JOIN_STATS
+    uint64_t getTS()
+    {
+        return GCToOSInterface::QueryPerformanceCounter();
```

Comment:
GetCycleCount32() is AMD64 assemby.  Prefer GCToOSInterface::QueryPerformanceCounter().  Centralize by using getTS()